src/libraries/Microsoft.Win32.SystemEvents/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0"?>
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:Microsoft.Win32.PowerModeChangedEventArgs</Target>
+    <Left>lib/netstandard2.0/Microsoft.Win32.SystemEvents.dll</Left>
+    <Right>lib/net461/Microsoft.Win32.SystemEvents.dll</Right>
```

Comment:
I suspect these are because of typeforwards. We could simplify this suppressions file by just adding a "global" suppression for this tuple, without a specific target API. i.e:

```xml
<?xml version="1.0"?>
<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <Suppression>
   <DiagnosticId>CP0001</DiagnosticId>
    <Left>lib/netstandard2.0/Microsoft.Win32.SystemEvents.dll</Left>
    <Right>lib/net461/Microsoft.Win32.SystemEvents.dll</Right>
  </Suppression>
</Suppressions>
```

Reply:
Same for all other suppression files where we want to suppress this diagnostic ID for all types in a specific tuple.

Reply:
The PackageValidation used has [the change](https://github.com/dotnet/sdk/commit/698ab94e401febe49d836ebfc331c6e8bb17f19f) but it doesn't work.

```xml
<?xml version="1.0" encoding="utf-8"?>
<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <Suppression>
    <DiagnosticId>CP0001</DiagnosticId>
    <Left>lib/netstandard2.0/System.IO.Ports.dll</Left>
    <Right>lib/net461/System.IO.Ports.dll</Right>
  </Suppression>
</Suppressions>
```

```
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.Handshake' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.Parity' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialData' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialDataReceivedEventArgs' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialDataReceivedEventHandler' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialError' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialErrorReceivedEventArgs' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialErrorReceivedEventHandler' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPinChange' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]       
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPinChangedEventArgs' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPinChangedEventHandler' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPort' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.StopBits' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.Handshake' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.Parity' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialData' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialDataReceivedEventArgs' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialDataReceivedEventHandler' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialError' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialErrorReceivedEventArgs' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialErrorReceivedEventHandler' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPinChange' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]       
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPinChangedEventArgs' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPinChangedEventHandler' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPort' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.StopBits' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.Handshake' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.Parity' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialData' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialDataReceivedEventArgs' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialDataReceivedEventHandler' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialError' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialErrorReceivedEventArgs' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialErrorReceivedEventHandler' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPinChange' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]       
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPinChangedEventArgs' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPinChangedEventHandler' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPort' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.StopBits' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.Handshake' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.Parity' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialData' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialDataReceivedEventArgs' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialDataReceivedEventHandler' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialError' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialErrorReceivedEventArgs' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialErrorReceivedEventHandler' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPinChange' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPinChangedEventArgs' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPinChangedEventHandler' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPort' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.StopBits' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.Handshake' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.Parity' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialData' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialDataReceivedEventArgs' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialDataReceivedEventHandler' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialError' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialErrorReceivedEventArgs' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialErrorReceivedEventHandler' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPinChange' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPinChangedEventArgs' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPinChangedEventHandler' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPort' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.StopBits' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.Handshake' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.Parity' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialData' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialDataReceivedEventArgs' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialDataReceivedEventHandler' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialError' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialErrorReceivedEventArgs' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialErrorReceivedEventHandler' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPinChange' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]       
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPinChangedEventArgs' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPinChangedEventHandler' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.SerialPort' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
C:\Users\vihofer\.nuget\packages\microsoft.dotnet.packagevalidation\1.0.0-rc.1.21376.22\build\Microsoft.DotNet.PackageValidation.targets(22,5): error CP0001: Type 'System.IO.Ports.StopBits' exists on runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll but not on lib/net461/System.IO.Ports.dll [C:\git\runtime3\src\libraries\System.IO.Ports\src\System.IO.Ports.csproj]
```

Reply:
I see another left listed on the errors, that is comparing the Unix runtime for ns2.0 agains the net461 lib.

Reply:
Is that a bug in PackageValidation or in the System.IO.Ports package?

Reply:
@Anipik is the comparison of those two assets expected? Having the runtime asset at the left compared against the net461 lib?

Reply:
It is weird that the package has a ns2.0/Unix runtime asset. @ericstj is that correct?

Reply:
It's fine to have a RID-specific netstandard asset.  That just means that the managed surface area used by the library can be satisfied by netstandard, but it needed to depend on some platform specific API.

I do find it odd that PackageValidation is asserting that `runtimes/unix/lib/netstandard2.0` asset needs to be compatible with `lib/net461` that seems like a bug.

Reply:
> I do find it odd that PackageValidation is asserting that runtimes/unix/lib/netstandard2.0 asset needs to be compatible with lib/net461 that seems like a bug.

Yeah that seems pretty weird.

Reply:
It seems like this was fixed with the latest package.